src/realm/exec/CMakeLists.txt

```diff
@@ -24,14 +24,14 @@ target_link_libraries(RealmTrawler Storage)
 
 add_executable(RealmBrowser EXCLUDE_FROM_ALL realm_browser.cpp )
 set_target_properties(RealmBrowser PROPERTIES
-    OUTPUT_NAME "realm-browser-6"
+    OUTPUT_NAME "realm-browser-10"
```

Comment:
Can we just remove these suffixes? What value are they adding?

Reply:
They are adding the value that It is build into the name which version it can handle. I have to have multiple versions around so that I can open whatever realm file I get from a customer.

Reply:
We currently just pin to a specific version (though we probably should be able to pick up the latest merge), but once this goes in I can update the version / name of the executable that we link to in our evergreen config. Thanks for pointing out